Extended guidelines :1984: (#8)
mirukucafe/legal: guidelines.md

1. No spamming. Spamming is the repeated uploading of the same or similar content over and over for where there is no real effort put into the content.

> 2. Do not post NSFW or/of suggestive minors, including drawings; **"B-But fiction doesn't affect reality"** well it seems like its affecting the reality of your cock and balls.

Reviewer: ```suggestion
2. Do not post NSFW or/of suggestive minors, including drawings
```
I believe you did a little bit of trolling :trollface:

Author: > I believe you did a little bit of trolling :trollface:

:nerd_face:

3. No racism, sexism, homophobia, transphobia, xenophobia, or casteism; do not participate in hate speech, violent extremism; do not promote self-harm or suicide; do not harass users or evade blocks/bans; No harassment, dogpiling or doxxing of other users

4.  Do not share malware, or any viruses; no sharing illegal content such as pirated content, and more.

> 5. Follow the [Terms of Service](https://github.com/mirukucafe/legal/blob/main/terms_of_service.md) for [miruku.cafe](https://miruku.cafe/@mclnoot) or else... (Also, the terms of service for miruku and this guideline is not in github gist so cry about it :trollface:)

Reviewer: ```suggestion
5. Follow our [Terms of Service](https://github.com/mirukucafe/legal/blob/main/terms_of_service.md) for [miruku.cafe](https://miruku.cafe/@mclnoot) or else... (Also, McInoot is based so cry about it)
```

Author: > 

McInoot :skull: :skull: :skull:

Author: > 

to https://gist.github.com/FanMclaine/15cd6314610415ed340e5391defe5469 you go
